src/plugins/vis_type_timeseries/server/routes/post_vis_schema.ts

```diff
@@ -247,6 +247,7 @@ export const visPayloadSchema = schema.object({
       series: schema.arrayOf(seriesItems),
       show_grid: numberIntegerRequired,
       show_legend: numberIntegerRequired,
+      tooltip_mode: stringOptionalNullable,
```

Comment:
Instead of allowing any string here, could we please make this just specific for the options? `schema.nullable(schema.oneOf(['show_all', 'show_focused']))`